### ISSUE TITLE
Midas Touch/Seeker Eclipse crash

### ISSUE THREAD

**Issue by SirHaesslich** [gskleres/FruityMod-StS#198]
Exhausting the card with Midas Touch and getting an Eclipse in return resulted in a crash

This can possibly be due to RPTS also having a card named Eclipse, I cannot tell from the log, if it is the case, I am sorry for blaming this on you, but it will be a good thing to know either way, and not only for me.

**Comment by fiiiiilth** [comment]
Yeah, the Eclipse from FruityMod and ReplayTheSpire had the same ID, which was causing issues. 

I already have a fix for this coming soon.
